openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -169,6 +154,18 @@ export const pluralize = (count: number, noun: string, suffix = 's') => {
   }
 };
 
+export const pluralizeWord = (count: number, noun: string, suffix = 's') => {
```

Comment:
We have a `pluralize` function, can we use that instead of creating new one?
https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx#L157